### DISCUSSION TITLE
flask db upgrade hangs on remote server

### DISCUSSION THREAD

**Discussion by PhenixDhinesh** [miguelgrinberg/Flask-Migrate#508] [Q&A]
i am working on a project and working with flask migrate also...i have few migrations and those are working well and i bought a free mysql database in db4free to deploy the model. and the deployed application is working well. but when i tried to do flask db upgrade on remote server.....that hangs on there.even though the migrate command detects all changes and creates the file.

**Comment by miguelgrinberg** [comment]
I'm sorry but I cannot help with this. Flask-Migrate is a very thin wrapper that makes Alembic accessible to Flask applications. Nothing in this package talks to the database directly. You should enable logging in SQLAlchemy to see what commands it is issuing and then correlate them with logs from your database to see what is happening.
